charts/spire/charts/tornjak-frontend/templates/deployment.yaml

```diff
@@ -35,6 +35,12 @@ spec:
           env:
             - name: REACT_APP_API_SERVER_URI
               value: {{ include "tornjak-frontend.apiURL" . }}
+            {{- if eq (.Values.spireHealthCheck.enabled | toString) "true" }}
+            - name: REACT_APP_SPIRE_HEALTH_CHECK_ENABLE
+              value: "{{ .Values.spireHealthCheck.enabled }}"
+            - name: REACT_APP_SPIRE_HEALTH_CHECK_FREQ_SEC
+              value: "{{ .Values.spireHealthCheck.frequencySeconds }}"
+            {{- end }}
```

Comment:
Maybe rename these values to follow the conventions of a k8s deployment as defined on line 48-52 in this file. Don't know if we should have spire in the naming of these values 🤔 

maybe something like `.Values.healthchecks.periodSeconds`.

I might also be confusing these with the k8s healthchecks as I can't really follow how these healtchecks are used in the frontend and what they are really checking for.

Reply:
I can rename the Helm variable per your suggestion, but we still have to keep the word `spire` in the name as the Tornjak API executes a call to SPIRE APIs to run SPIRE server health-check. see: https://github.com/spiffe/spire/blob/aefb6b487dcb23962536ce5603df53e61fe35efa/cmd/spire-server/cli/healthcheck/healthcheck.go#L55
Without `spire` it implies this is a Tornjak health-check command.